ote/src/clj/ote/services/transport.clj

```diff
@@ -164,13 +174,16 @@
                                ::t-operator/type            "organization"
                                ::t-operator/approval_status "approved"
                                ::t-operator/is_organization true})
-        member (specql/insert! db ::user/member
-                               {::user/id         (str (UUID/randomUUID))
-                                ::user/table_id   (get-in user [:user :id])
-                                ::user/group_id   (:ote.db.transport-operator/group-id group)
-                                ::user/table_name "user"
-                                ::user/capacity   "admin"
-                                ::user/state      "active"})]
+        member (create-member! db user-id group)
```

Comment:
Would removing the sql `WHERE` condition "```AND u.id != :user-id```" allow removing this separate adding of user to own organisation, would the loop below then add also himself?

Reply:
I had a reason to do it this way, but now when I look at this it seems stupid and I don't remember why I needed to do this using loop for others and direct call for the creator. Jeah. I'll change this. Thanx